elrond-wasm-node/src/api/send_api_node_impl_managed.rs

```diff
@@ -398,49 +406,9 @@ impl SendApiImpl for VmApiImpl {
                 result_handle,
             );
 
-            ManagedVec::from_raw_handle(result_handle)
-        }
-    }
+            self.clean_return_data();
```

Comment:
I would like us to keep the API traits as close to the actual VM as possible.
That is why I would prefer to have this called higher-level, in the ContractCall, ContractDeploy and SendWrapper.
Probably the best way to implement would be to have an `execute_on_dest_context_raw` method in the SendWrapper and have ContractCall, ContractDeploy call it (this is some technical debt we need to sort out - the send api is doing too much).

On a related note: forwarder-raw is calling the API directly. This is also a problem, we should discourage this, it should also go though `self.send().execute_on_dest_context_raw(...)`.

Another technical debt: `call_local_esdt_built_in_function` does not belong in the API.